src/client/swagger/unity.yml

```diff
@@ -774,3 +864,140 @@ components:
+          oneOf:
+            - $ref: '#/components/schemas/RestrictedLimit'
+            - type: integer
+              minimum: 1
+              maximum: 1000000
+    BucketLimits:
+      description: Bucket limits
+      type: object
+      properties:
+        maxBuckets:
+          example: 2
+          description: Number of buckets allowed
+          oneOf:
+            - $ref: '#/components/schemas/RestrictedLimit'
+            - $ref: '#/components/schemas/Unlimited'
+            - type: integer
+          minimum: 1
+        maxRetentionDuration:
+          description: Retention duration limits in nanoseconds
+          example: 2592000000000000
+          oneOf:
+            - $ref: '#/components/schemas/RestrictedLimit'
+            - $ref: '#/components/schemas/Unlimited'
+            - type: integer
+          minimum: 1
+    TaskLimits:
+      description: Task limits
+      type: object
+      properties:
+        maxTasks:
+          description: Number of tasks allowed
+          example: 5
+          oneOf:
+            - $ref: '#/components/schemas/RestrictedLimit'
+            - $ref: '#/components/schemas/Unlimited'
+            - type: integer
+          minimum: 1
+    DashboardLimits:
+      description: Dashboard limits
+      type: object
+      properties:
+        maxDashboards:
+          description: Number of dashboards allowed
+          example: 5
+          oneOf:
+            - $ref: '#/components/schemas/RestrictedLimit'
+            - $ref: '#/components/schemas/Unlimited'
+            - type: integer
+          minimum: 1
+    CheckLimits:
+      description: Check limits
+      type: object
+      properties:
+        maxChecks:
+          description: Number of checks allowed
+          example: 2
+          oneOf:
+            - $ref: '#/components/schemas/RestrictedLimit'
+            - $ref: '#/components/schemas/Unlimited'
+            - type: integer
+          minimum: 1
+    NotificationRuleLimits:
+      description: Notification rule limits
+      type: object
+      properties:
+        maxNotifications:
+          description: Number of notifications allowed
+          example: 2
+          oneOf:
+            - $ref: '#/components/schemas/RestrictedLimit'
+            - $ref: '#/components/schemas/Unlimited'
+            - type: integer
+          minimum: 1
+        blockedNotificationRules:
+          description: Notification rules not allowed CSV
+          example: 'http,pagerduty'
+          type: string
+    NotificationEndpointLimits:
+      description: Notification endpoint limits
+      type: object
+      properties:
+        blockedNotificationEndpoints:
+          description: Notification endpoints not allowed CSV
+          example: 'http,pagerduty'
+          type: string
+    RestrictedLimit:
+      description: Restrict access entirely
+      type: integer
+      enum: [-1]
+    Unlimited:
+      description: Unbounded limit
+      type: integer
+      enum: [0]
```

Comment:
Are the Limits necessary? The UI currently gets that data from IDPE

Reply:
We'll need them for the operator UI because it provides the ability to change limits for a user. A user wouldn't have permission to change their own limits.

Reply:
Gotcha.  I think i bundled this up with the org limits stuff in the other comment. That makes sense, not entirely certain of the dynamic but it seems unique